app/src/main/java/com/sunlightlabs/congress/models/Legislator.java

```diff
@@ -40,6 +40,37 @@ public String getOfficialName() {
 		return last_name + ", " + firstName();
 	}
 
+	public static String[] splitName(String displayName) {
+		String[] pieces = displayName.split(" ");
+		String first_name = pieces[0];
+		String last_name = pieces[pieces.length-1];
+
+		if (
+			last_name.equals("Jr.") ||
+			last_name.equals("II") ||
+			last_name.equals("III")
+		)
+			last_name = pieces[pieces.length-2] + " " + pieces[pieces.length-1];
```

Comment:
These are the 3 suffixes I'm attempting to accommodate for when doing string splitting to try to get the `first_name` and `last_name` fields out of a display name. @dwillis If you know of any others offhand, please let me know.

Related issue: https://github.com/propublica/congress-api-docs/issues/46